airbyte-integrations/connectors/source-shopify/source_shopify/utils.py

```diff
@@ -310,7 +310,7 @@ def stream_state_to_tmp(*args, state_object: Dict = cached_state, **kwargs) -> D
         # Map the input *args, the sequece should be always keeped up to the input function
         # change the mapping if needed
         stream: object = args[0]  # the self instance of the stream
-        current_stream_state: Dict = kwargs["stream_state"] or {}
+        current_stream_state: Dict = kwargs.get("stream_state") or {}
```

Comment:
Maybe `kwargs.get("stream_state", {})` would be better?

Reply:
It doesn't really matter, but I'll update as you suggest.